docs/api/web-contents.md

```diff
@@ -2064,6 +2064,21 @@ Setting the WebRTC IP handling policy allows you to control which IPs are
 exposed via WebRTC. See [BrowserLeaks](https://browserleaks.com/webrtc) for
 more details.
 
+#### `contents.getWebRTCUDPPortRangePolicy()`
+
+Returns `Object`:
+
+* `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
+* `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.
```

Comment:
Poking around in Chromium internals, it sounds like these would default to `{ min: 0, max: 0 }` which would apply no restriction on port range. Should we include a note (and maybe a unit test) about the default value/behavior of `setWebRTCUDPPortRangePolicy` hasn't been called?

Reply:
Yeah, a note for the default behavior is reasonable